--- a/dist/shader-mount.d.ts
+++ b/dist/shader-mount.d.ts
@@ -6,7 +6,7 @@
  * We just skip setting the uniform if it's undefined. This allows the shader mount to still take up space during server rendering
  */
 interface ShaderMountUniformsReact {
-    [key: string]: string | boolean | number | number[] | number[][] | HTMLImageElement | undefined;
+    [key: string]: string | boolean | number | number[] | number[][] | Float32Array | HTMLImageElement | undefined;
 }
 export interface ShaderMountProps extends Omit<React.ComponentProps<'div'>, 'color' | 'ref'>, ShaderMotionParams {
     ref?: React.Ref<PaperShaderElement>;
